### PR TITLE
Release notes

### PR DESCRIPTION
**Pull request by borditamas**

### CODE REVIEW

File: arrowhead-certificate-generator/downloads/release-notes-v1.0.0.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+RELEASE NOTES
+
+Arrowhead-Certificate-Generator-Setup-1.0.0
+03.11.2022
+
+Initial release:
+
+Easily and efficiently generate arrowhead certificates, whether for existing or newly installed clouds. The application guides you step-by-step through the process and
```
Comment (rbocsi) [inline arrowhead-certificate-generator/downloads/release-notes-v1.0.0.txt:8]: generates

File: arrowhead-certificate-generator/downloads/release-notes-v1.0.0.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+RELEASE NOTES
+
+Arrowhead-Certificate-Generator-Setup-1.0.0
+03.11.2022
+
+Initial release:
+
+Easily and efficiently generate arrowhead certificates, whether for existing or newly installed clouds. The application guides you step-by-step through the process and 
+ensures that the end result meets all the criteria
```
Comment (rbocsi) [inline arrowhead-certificate-generator/downloads/release-notes-v1.0.0.txt:9]: missing dot

File: arrowhead-framework-installer/downloads/release-notes-v1.0.0.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+RELEASE NOTES
+
+Arrowhead-Framework-Installer-1.0.0
+14.11.2022
+
+Initial release:
+
+Install the desired arrowhead central system with just a few clicks on the selected devices. Available in both online and offline versions, you can create your own local
```
Comment (rbocsi) [inline arrowhead-framework-installer/downloads/release-notes-v1.0.0.txt:8]: arrowhead => Arrowhead

File: arrowhead-framework-uninstaller/downloads/release-notes-v1.0.0.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+RELEASE NOTES
+
+Arrowhead-Framework-Uninstaller-1.0.0
+14.11.2022
+
+Initial release:
+
+Remove easily the selected arrowhead core systems which were installed by the Arrowhead-Framework-Installer from Aitia International Inc.
```
Comment (rbocsi) [inline arrowhead-framework-uninstaller/downloads/release-notes-v1.0.0.txt:8]: arrowhead =>Arrowhead

**Review by Aitia-IIOT — APPROVED** [review]

**Pull request merged**